addon/helpers/choice.js

```diff
@@ -9,6 +9,11 @@ export default MicroState.extend({
       return Type.create(values, options);
     },
 
+    set(choice, values, options) {
```

Comment:
I think set is _exceedingly_ unlikely for choice. If you want the choices to change, then more likely you would change the data that flowed into it. 

In the case where you _do_ want to call `set` then I think it's not unreasonable to require you to understand what a choice is and how to construct it. When you use `Number#set` you're expected to understand how to get a number in there. Same thing with `Choice`. It's because this is hard that `set` is not very likely to be called.

Let's just remove the `set` operation entirely for now.